Library/Homebrew/os/mac/xcode.rb

```diff
@@ -267,7 +267,7 @@ module CLT
       # The original Mavericks CLT package ID
       EXECUTABLE_PKG_ID = "com.apple.pkg.CLTools_Executables"
       MAVERICKS_NEW_PKG_ID = "com.apple.pkg.CLTools_Base" # obsolete
-      PKG_PATH = "/Library/Developer/CommandLineTools"
+      PKG_PATH = `/usr/bin/xcode-select -p`.chop.freeze
```

Comment:
Hard-coding `/Library/Developer/CommandLineTools` is intended here.

Reply:
why do?

Reply:
Because the [`CLT` module](https://github.com/Homebrew/brew/blob/ed793811553eb6fca3dd602931e37b7187f86a68/Library/Homebrew/os/mac/xcode.rb#L262-L270), which is where this code lives, is designed to find CLT installations via `xcode-select --install`. These are always installed in `/Library/Developer/CommandLineTools`. In particular, we don't want the `CLT` module to pick up Xcode installations, which are handled by the [`Xcode` module](https://github.com/Homebrew/brew/blob/ed793811553eb6fca3dd602931e37b7187f86a68/Library/Homebrew/os/mac/xcode.rb#L9).

Reply:
> Because the [`CLT` module](https://github.com/Homebrew/brew/blob/ed793811553eb6fca3dd602931e37b7187f86a68/Library/Homebrew/os/mac/xcode.rb#L262-L270), which is where this code lives, is designed to find CLT installations via `xcode-select --install`. These are always installed in `/Library/Developer/CommandLineTools`. In particular, we don't want the `CLT` module to pick up Xcode installations, which are handled by the [`Xcode` module](https://github.com/Homebrew/brew/blob/ed793811553eb6fca3dd602931e37b7187f86a68/Library/Homebrew/os/mac/xcode.rb#L9).

Unfortunately this is a bit of problem as if the xcode-select path is set to the xcode installation which provides the xcode command line tools, it breaks formula checking the existence of command line tools

Reply:
I don't see the value of hardcoding this path as it can be changed using `xcode-select -s`, which I am sure that you aware of.

Reply:
If there is a better way to fix this problem inform of it because I don't see how hardcoding it is better

Reply:
> Unfortunately this is a bit of problem as if the xcode-select path is set to the xcode installation which provides the xcode command line tools, it breaks formulas checking the existence of command line tools, such python and llvm.

Actually, that's another thing your proposed change breaks. When `llvm` and `python@3.*` check for the existence of the CLT, they are actually looking specifically for one installed at `/Library/Developer/CommandLineTools`, and not just any Xcode CLT installation.

If the `pour_bottle?` check is not needed or overly strict, we can discuss changing those (in their respective formulae), but what we shouldn't do is make a breaking change to the behaviour of the `CLT` module.